src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentResourceIT.java

```diff
@@ -285,59 +285,12 @@ public void shouldRetrieveAPayment_fromPublicApiEndpoint() {
                 .then()
                 .statusCode(OK.getStatusCode())
                 .contentType(JSON)
-                .body(JSON_CHARGE_KEY, is(paymentFixture.getExternalId()))
                 .body(JSON_AMOUNT_KEY, isNumber(paymentFixture.getAmount()))
                 .body(JSON_REFERENCE_KEY, is(paymentFixture.getReference()))
                 .body(JSON_DESCRIPTION_KEY, is(paymentFixture.getDescription()))
                 .body(JSON_STATE_STATUS_KEY, is(paymentFixture.getState().toExternal().getStatus()))
                 .body(JSON_STATE_FINISHED_KEY, is(false))
-                .body(JSON_STATE_DETAILS_KEY, is("example_details"))
-                .body(JSON_RETURN_URL_KEY, is(mandateFixture.getReturnUrl()));
```

Comment:
the fields JSON_RETURN_URL_KEY, JSON_CHARGE_KEY and FRONTEND_CARD_DETAILS_URL can be deleted. imports can be optimized on this class too.

Reply:
this isn't resolved yet as https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentResourceIT.java#L75, https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentResourceIT.java#L67-L68 and https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentResourceIT.java#L62 can be deleted. There are still some unused imports on this class.